lib/cf-socket.c

```diff
@@ -788,6 +788,7 @@ struct cf_socket_ctx {
 #endif
   BIT(got_first_byte);               /* if first byte was received */
   BIT(accepted);                     /* socket was accepted, not connected */
+  BIT(sock_connected);               /* socket is "connected", e.g. in UDP */
```

Comment:
I did not spot any *user* of this info, so I presume this is something you plan to use coming up?

Reply:
I found it interesting to keep, since the OpenSSL issue with connected socket requires, for now, a workaround. But we can also remove that in this PR and shift it to later.

Reply:
I just wanted to make sure. Let's keep it there for now.